test/kino_flame/runner_cell_test.exs

```diff
@@ -93,6 +172,16 @@ defmodule KinoFLAME.RunnerCellTest do
     assert source =~ "min: 5"
   end
 
+  test "sets missing_livebook_cookie if env var is missing" do
+    {kino, _source} = start_smart_cell!(RunnerCell, %{})
+
+    # missing helper in Kino.Test
+    # See https://github.com/livebook-dev/kino/pull/468
```

Comment:
```suggestion
    # TODO: use push_smart_cell_editor_source once released https://github.com/livebook-dev/kino/pull/468
    # (no need to bump :kino requirement, because it's test-only)
```